### PR TITLE
[이원규-4주차 알고리즘 스터디]

### PR DESCRIPTION
**Pull request by OneK-2**
### Week 04 (24.08.08 ~ 24.08.14)
| Category | Title | Link |
| :------: | :---: | :--: |
| 구현 | <a href="https://www.acmicpc.net/problem/20436">ZOAC 3</a> | <a href="https://github.com/Algorithm-Study-AS/Algorithm-study/pull/15/commits/993aa40d41e66f87a91f2fd2a836812eae8cdee8">🔗</a> |
| 구현 | <a href="https://www.acmicpc.net/problem/20164">홀수 홀릭 호석</a> | <a href="홀수 홀릭 호석https://github.com/Algorithm-Study-AS/Algorithm-study/commit/ac04e422581a03dae1426afabcbc65e3f5858fc0">🔗</a> |
| DP | <a href="https://www.acmicpc.net/problem/2156">포도주 시식</a> | <a href="https://github.com/Algorithm-Study-AS/Algorithm-study/commit/bda8d248fc004c6d59fdf9456be7a0f417f5a1aa">🔗</a> |
| DP | <a href="https://www.acmicpc.net/problem/9655">돌 게임</a> | <a href="https://github.com/Algorithm-Study-AS/Algorithm-study/commit/abb854250188a6a1d3189b40e192a2bb1cfefcc2">🔗</a> |
<br>

```
To. Reviewers
```

### CODE REVIEW

File: 이원규/DP/포도주 시식-2156.java
Comment (harin1212) [inline 이원규/DP/포도주 시식-2156.java:1]: 저도 비슷하게 풀었습니다!

File: 이원규/구현/ZOAC 3-20436.java
Comment (harin1212) [inline 이원규/구현/ZOAC 3-20436.java:1]: 저도 이 코드 시간 궁금합니다!

File: 이원규/구현/홀수 홀릭 호석-20164.java
Comment (harin1212) [inline 이원규/구현/홀수 홀릭 호석-20164.java:1]: 동일하게 풀이했습니다

**Review by harin1212 — APPROVED** [review]
이번주도 고생하셨습니다!!

File: 이원규/구현/ZOAC 3-20436.java
Comment (OneK-2) [reply 이원규/구현/ZOAC 3-20436.java:1]: 100ms 나왔습니다!

**Pull request merged**